extensions/adobe/experience/adcloud/experienceevent-all.schema.json

```diff
@@ -40,6 +40,21 @@
           "title": "Stitch ID Chosen",
           "type": "string",
           "description": "ID from the Ad Servers through AdCloud STATS to track click-through conversion on browsers that block third party cookies"
+        },
+        "https://ns.adobe.com/experience/adcloud/adDeliveryDetails": {
+          "title": "Ad Delivery Details",
+          "$ref": "https://ns.adobe.com/experience/adcloud/addeliverydetails",
+          "description": "Adobe AdCloud Search Ad Delivery Details."
+        },
+        "https://ns.adobe.com/experience/adcloud/productDetails": {
+          "title": "Ad Product Details",
+          "$ref": "https://ns.adobe.com/experience/adcloud/productdetails",
+          "description": "Adobe AdCloud Search Ad Product Details."
+        },
+        "https://ns.adobe.com/experience/adcloud/conversionDetails": {
+          "title": "Ad Conversion Details",
+          "$ref": "https://ns.adobe.com/experience/adcloud/conversiondetails",
+          "description": "Search Ad Conversion Details."
         }
       }
     }
```

Comment:
Please remove
"$ref": "https://ns.adobe.com/xdm/context/experienceevent#/definitions/experienceevent"

The experience event field will be added by the schema that is based on the class